client/src/theme/variables.css

```diff
@@ -777,7 +777,6 @@ body {
 ::-webkit-scrollbar {
   width: 1em;
   border-radius: 1em;
-
```

Comment:
Why backgrounds were remove here btw?

Reply:
Tests by @fabienSvtr, I reverted the changes.